ncm-iptables/src/main/perl/iptables.pod

```diff
@@ -10,15 +10,11 @@ iptables: Setup the IPTABLES firewall rules.
 =head1 DESCRIPTION
 
 The I<IPTABLES> component perform the setup of the
-B</etc/sysconfig/iptables> configuration file and restarts the
+C<< /etc/sysconfig/iptables >> configuration file and restarts the
 iptables service.
 
 =head1 SYNOPSIS
 
-Note: a detailed HOWTO for this component, including examples, is
-found in /usr/share/doc/ncm-iptables-2.3.13/ncm-iptables-howto.html and in the FIO twiki pages
```

Comment:
if we can get the old rpm, I'm happy to read anything that is in it. maybe it also still contains the 2 missing pan/tpl files.